projects/storefrontlib/shared/components/captcha/mockCaptcha/config/captcha-api-config.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, Type } from '@angular/core';
+import { Config } from '@spartacus/core';
+import { CaptchaProvider } from '../../captcha.model';
+
+@Injectable({
+  providedIn: 'root',
+  useExisting: Config,
+})
+export abstract class CaptchaApiConfig {
+  captchaProvider?: Type<CaptchaProvider>;
```

Comment:
This file should be a part of template, maybe move it from mock folder to projects/storefrontlib/shared/components/captcha/config/captcha-api-config.ts

Reply:
fixed